docs/using-airbyte/getting-started/readme.md

```diff
@@ -2,45 +2,42 @@
 :::
 
-## Sign Up for Airbyte Cloud
+## Airbyte Cloud
 
-To use Airbyte Cloud, [sign up](https://cloud.airbyte.io/signup) with your email address, Google login, or GitHub login. Upon signing up, you'll be taken to your workspace, which lets you collaborate with team members and share resources across your team under a shared billing account.
+To use Airbyte Cloud, [sign up](https://cloud.airbyte.io/signup) with your email address, Google login, or GitHub login. Upon signing up, you'll be taken to your workspace. There, you can collaborate with team members, sharing resources across your team under a single billing account.
 
-Airbyte Cloud offers a 14-day free trial that begins after your first successful sync. For more details on our pricing model, see our [pricing page](https://www.airbyte.com/pricing).
+Airbyte Cloud offers a 14-day free trial that begins after your first successful sync. You can find more details about Airbyte Cloud for practitioners and teams, as well as a tool for evaluating costs on our [pricing page](https://www.airbyte.com/pricing).
 
 To start setting up a data pipeline, see how to [set up a source](./add-a-source.md).
 
-## Deploy Airbyte (Self-Managed)
-
-When self-managing Airbyte, your data never leaves your premises. Get started immediately by deploying locally using Docker.
+## Open Source Community (OSS) 
 
-### Self-Managed Community (Open Source)
+When self-managing Airbyte, your data never leaves your premises. You can use our [OSS Quickstart](oss-quickstart.md) to get started immediately by deploying locally using Docker. 
 
-With Airbyte Self-Managed Community (Open Source), you can use one of the following options in your infrastructure:
+:::tip
+If you're trying Airbyte out for the first time, we'd recommend you sign up for the [Airbyte Cloud trial](https://cloud.airbyte.io/signup) or deploy OSS locally.
+:::
 
-- [Local Deployment](/deploying-airbyte/quickstart.md) (recommended when trying out Airbyte)
-- [On AWS](/deploying-airbyte/on-aws-ec2.md)
-- [On Azure VM Cloud Shell](/deploying-airbyte/on-azure-vm-cloud-shell.md)
-- [On Digital Ocean Droplet](/deploying-airbyte/on-digitalocean-droplet.md)
-- [On GCP](/deploying-airbyte/on-gcp-compute-engine.md)
-- [On Kubernetes](/deploying-airbyte/on-kubernetes-via-helm.md)
-- [On OCI VM](/deploying-airbyte/on-oci-vm.md)
-- [On Restack](/deploying-airbyte/on-restack.md)
-- [On Plural](/deploying-airbyte/on-plural.md)
-- [On AWS ECS](/deploying-airbyte/on-aws-ecs.md) (Spoiler alert: it doesn't work)
+If your use case calls for it, you can also deploy our OSS. When you're ready to deploy, refer to our documentation on deploying Airbyte. The recommended method for this is on [Kubernetes via Helm](../../deploying-airbyte/on-kubernetes-via-helm.md). 
```

Comment:
If your use case calls for it, you can also deploy our OSS. When you're ready to deploy, refer to our documentation on deploying Airbyte. The recommended method for this is on [Kubernetes via Helm](../../deploying-airbyte/on-kubernetes-via-helm.md). -> To deploy Airbyte in production, Airbyte recommends deploying on [Kubernetes via Helm](../../deploying-airbyte/on-kubernetes-via-helm.md).